src/main_prog.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+
+import material_properties
+
+import generate_mesh
+
+# Allow for user input for number of materials and what kind of materials
+
+parser = argparse.ArgumentParser()
+    
+parser.add_argument("-n", "--number_materials", type=int, default=3, required=False,
+                           help="Determines how many materials are in the particle")
+
+
+parser.add_argument("-f", "--fuel", type=list(tuple), default=[('UO2',250e-6)], required=False,
+	               choices=[('UO2',250e-6),('UF4',250e-6)],
+	               help="Determine the type of fuel used in the particle and its radius")
+
+parser.add_argument("-c", "--clad", type=list(tuple), default=[('Inner Pyrolytic Carbon', 35e-6) , ('Silicon Carbide', 35e-6)],
```

Comment:
Perhaps a similar comment to the one above, although the choices here are not constrained by a `choices` parameter.  Nevertheless, you might want to populate a set of choices for the material from your material properties library.

Reply:
That is a good idea. Do you have a suggestion on how to do so? The material properties library is currently inside the material properties function which takes an input argument of the material information. The arguments are used to build the material information. So, I am unsure how to call the material properties function without the material information.

Reply:
`choices=material_properties.material_props_master.keys()`